README.markdown

```diff
@@ -21,7 +21,7 @@ Build and Install
 The build install instructions are currently being written. The Xapi
 Project contains a large list of dependencies and sub-projects, which
 are actually quite difficult to build independently. To build xen-api
-from source, we recommend using [xenserver-core](https://github.com/xenserver/xenserver-core).
+from source, we recommend using [dune (previously named jbuilder)](https://github.com/ocaml/dune)
```

Comment:
You have to use dune as the build tool, but that would not be enough. I think here we should say opam using he remote provided by xs-opam, and link to the xs-opam readme